handler.js

```diff
@@ -193,16 +201,38 @@ async function approve(responseUrl, emailAtt, user) {
       ReplyToAddresses: [tmplContext.author_name],
       Message: {
         Subject: {
-          Data: emailAtt.title
+          Data: emailSubject
         },
         Body: {
           Text: {
-            Data: EMAIL_TEMPLATE(tmplContext),
+            Data: emailBody,
             Charset: 'UTF-8'
           }
         }
       }
     }).promise();
+
+    if (JSON.parse(process.env.LOG_JSON_TO_S3.toLowerCase())) {
+      const logEntry = {
+        projectid: projectId,
+        from: tmplContext.author_name,
+        to: sendTo,
```

Comment:
This is somewhat minor, but `from` and `to` are reserved keywords in SQL, so escaping or remapping will be required in Athena. Maybe these could just be named something else, for convenience? Like `sender` and `recipients` or something like that?